src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -151,9 +151,13 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
 
         auto rep_file_name = ip->getAlphaFieldValue(fields, objectSchemaProps, "representation_file_name");
         fs::path rep_file_path = DataSystemVariables::CheckForActualFilePath(state, fs::path(rep_file_name), std::string(RoutineName));
+        if (rep_file_path.empty()) {
+            ErrorsFound = true;
+        }
```

Comment:
I guess the CheckForActualFilePath will emit warnings so that it is sufficient to just set the error flag to true here.